hack/remove-calico-policy/override-policy.md

```diff
@@ -19,17 +19,33 @@ ingress and egress traffic, overriding any other Network Policy that has been co
 
 #### Override Calico policy to allow all traffic
 
-Create a `GlobalNetworkPolicy` resource by running the following command:
+Create a `GlobalNetworkPolicy` (`SystemNetworkPolicy` for Calico `v2.4.x` or older) resource by running the following command:
+
+For Calico `v2.5.x` or higher:
 
 ```
 kubectl create -f=https://raw.githubusercontent.com/projectcalico/calico/master/hack/remove-calico-policy/global-network-policy-override.yaml
 ```
 
+For Calico `v2.4.x` or older:
+
+```
+kubectl create -n=kube-system -f=https://raw.githubusercontent.com/projectcalico/calico/master/hack/remove-calico-policy/system-network-policy-override.yaml
+```
+
 #### Revert override to enable Calico policy
 
-To revert the override of Calico policy, delete the `GlobalNetworkPolicy` resource 
+To revert the override of Calico policy, delete the `GlobalNetworkPolicy` (`SystemNetworkPolicy` for Calico `v2.4.x` or older) resource 
```

Comment:
again, perhaps easier here just to say - to remove the policy-override resource..